README.md

```diff
@@ -128,7 +128,8 @@ The file `controllers/common_test.go` exports functions commonly used in control
 - `middleware`: Contains all middleware.
 
 ### Endpoints
-TODO: Find a good, automated way to generate endpoint documentation. (See also: [File Structure](#file-structure))
+To write annotations for the endpoints, you can refer to [swag](https://github.com/swaggo/swag#readme)
```

Comment:
Change the order of these and also add an empty line in between since markdown doesn't render single line breaks.